CHANGELOG.md

```diff
@@ -1,26 +1,25 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
-NOTE: Realm file format is likely to break and you CAN NOT revert back to the previous file format - DATA WILL BE LOST!!!!
-NOTE: Sync protocol version 4: CANNOT SYNC WITH MONGODB REALM CLOUD.
-NOTE: Bump file format version to 21. NO DOWNGRADE PATH IS AVAILABLE.
+NOTE: Bump file format version to 22. NO DOWNGRADE PATH IS AVAILABLE.
 
 ### Enhancements
-* None.
+* Change listener for dictionary is not key based.
```

Comment:
Shouldn't this be noted under a "known issues" heading or similar?

Reply:
My guess is that he meant to write "now" instead of "not":
```suggestion
* Change listener for dictionary is now key based.
```

Reply:
Add link to PR or issue (that has a link to PR)

Reply:
Perhaps also mention that this is breaking the API introduced in beta "?"?

Reply:
I think I need to rephrase it to "Change listeners for dictionaries are now key based"